### PR TITLE
VDS-2022: Add 'Emails' column to import/export

### PR DESCRIPTION
**Pull request by ilyawzrd**
Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.CustomerExportImport_1.0.0-pr-41-071396a9.zip

### CODE REVIEW

**Review by pushnitsa — APPROVED** [review]

File: src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvOrganization.cs [hunk 1]
```diff
@@ -95,6 +96,7 @@ public void PatchOrganization(Organization target)
         {
             target.OuterId = OuterId;
             target.Name = OrganizationName;
+            target.Emails = string.IsNullOrEmpty(Emails) ? null : Emails.Split(", ");
```
Comment (asvishnyakov) [inline src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvOrganization.cs:99]: Replace with `.Split(',').Select(phone => phone.Trim()).ToList()`

File: src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvContact.cs [hunk 2]
```diff
@@ -191,6 +192,7 @@ public void PatchContact(Contact target)
             target.AssociatedOrganizations = (string.IsNullOrEmpty(AssociatedOrganizationIds) ? null : AssociatedOrganizationIds?.Split(", "))?.Select(x => x.Trim()).Where(x => !string.IsNullOrEmpty(x)).ToList();
             target.BirthDate = Birthday;
             target.TimeZone = TimeZone;
+            target.Emails = string.IsNullOrEmpty(Emails) ? null : Emails.Split(", ");
```
Comment (asvishnyakov) [inline src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvContact.cs:195]: Replace with `.Split(',').Select(phone => phone.Trim()).ToList()`

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvOrganization.cs (hunk 1 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvOrganization.cs]: done

File: src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvContact.cs (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.CustomerExportImportModule.Core/Models/CsvContact.cs]: done

**Review by asvishnyakov — APPROVED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**